File: Back/NodeJS/modules/bets/src/application/features/create-bet/BetCreatedNotification.ts
import { BetId } from "../../../domain/bets/BetId";
import { MemberId } from "../../../domain/members/MemberId";
import { INotification } from "../../../../../shared/application/Request/INotification";

export class BetCreatedNotification extends INotification {
    constructor(public BetId: BetId, public MemberId: MemberId, public Coins: number){
        super();
    }
    Name: string = BetCreatedNotification.name;
}

File: Back/.Net/Modules/Bets/BetFriends.Bets.Application/Features/CreateBet/BetCreatedNotification.cs
using BetFriends.Bets.Domain.Bets;
using BetFriends.Bets.Domain.Members;
using MediatR;

namespace BetFriends.Bets.Application.Features.CreateBet;

public record BetCreatedNotification(BetId BetId, MemberId OwnerId, int Coins) : INotification;